client/src/components/home/Home.tsx

```diff
@@ -0,0 +1,97 @@
+
+import { DefaultButton, Link, PartialTheme, StackItem, Theme } from '@fluentui/react';
+import { Stack, Text, Image } from '@fluentui/react';
+import imageCalendar from '../../assets/lightCalendarSymbol.png';
+import {
+  btnStackStyles,
+  buttonStyle,
+  containerStyles,
+  font12pxSemiBoldStyle,
+  font12pxWeight400Style,
+  font16pxStyle,
+  fullScreenStyles,
+  innerContainer,
+  lineHeight22px,
+  lineHeight28px
+} from '../../styles/Home.styles';
+import { FrequentlyAskedQuestions } from '../FrequentlyAskedQuestions';
+import { LearnMoreItem } from '../LearnMoreItem';
+
+export interface HomeProps {
+  companyName: string;
+  theme?: PartialTheme | Theme;
+}
+
+export const HomeComponent = (props: HomeProps): JSX.Element => {
+  return (
+    <Stack styles={fullScreenStyles}>
+      <Stack
+        horizontalAlign="center"
+        verticalAlign="start"
+        styles={containerStyles(props.theme)}
+        tokens={{ childrenGap: 15 }}
+      >
+        <Stack styles={innerContainer}>
+          <Stack verticalAlign="center" horizontalAlign="center">
+            <Image width="23.75rem" height="6.25rem" src={imageCalendar} alt="calendarImage"></Image>
+          </Stack>
+          <Stack>
+            <Text styles={lineHeight28px}>Hello,</Text>
+            <Text styles={lineHeight22px}>What would you like to do?</Text>
+            <Stack horizontal horizontalAlign="space-evenly" styles={btnStackStyles}>
+              <StackItem>
+                <DefaultButton
+                  text="Book an appointment"
+                  styles={buttonStyle}
+                  iconProps={{ iconName: 'Calendar' }}
+                  onClick={() => window.location.replace('/book')}
+                />
+              </StackItem>
+              <StackItem>
+                <DefaultButton
+                  text="Join from link"
+                  styles={buttonStyle}
+                  iconProps={{ iconName: 'Link' }}
+                  onClick={() => window.location.replace('/visit')}
+                />
+              </StackItem>
+            </Stack>
+            <FrequentlyAskedQuestions />
+            <Text styles={font16pxStyle}>Learn more about Azure Communication Services</Text>
+            <LearnMoreItem
+              headerText={'Azure Communication Services virtual appointments'}
+              headerLink={'https://learn.microsoft.com/en-us/azure/communication-services/tutorials/virtual-visits'}
+              description={
+                'This tutorial describes concepts for virtual appointment applications. After completing this tutorial and the associated Sample Builder, you will understand common use cases that a virtual appointment delivers'
+              }
+            />
+            <Link
+              styles={font12pxSemiBoldStyle}
+              href="https://learn.microsoft.com/en-us/samples/azure-samples/communication-services-dotnet-quickstarts/azure-communication-services---rooms-public-preview/"
```

Comment:
Let's remove the locale `en-us` from all of these links since users may come from different locations / use different languages

Reply:
done